hack/build.sh

```diff
@@ -81,7 +81,9 @@ run() {
 
 go_fmt() {
   echo "🧹 ${S}Format"
-  go fmt ./cmd/... ./pkg/...
+  gofmt -w $(find -name '*.go' | grep -v vendor)
+  echo "🧹 ${S}Format: Simplify code.."
```

Comment:
I think we should skip that log message (as well as below). The message it intentionally kept terse to not clutter the CLI. If you need more details you could always use `--verbose`. The overall "Format" message is good enough (even when it encompasses  two commands)

Reply:
resolved